markdown/postprocessors.py

```diff
@@ -75,9 +75,18 @@ def run(self, text):
                     self.md.htmlStash.get_placeholder(i))] = html
             replacements[self.md.htmlStash.get_placeholder(i)] = html
 
+        def substitute_match(m):
+            key = m.group(0)
+
+            if key not in replacements:
+                return '<p>' + replacements[key[3:-4]] + '</p>'
```

Comment:
Here it would be:

```python
return f'<p>{ replacements[key[3:-4]] }</p>' 
```

Reply:
Excellent. Thank you for the recommendation.